### DISCUSSION TITLE
Can't submit and weird rejection

### DISCUSSION THREAD

**Discussion by FrancyMak** [community/community#80567] [GitHub Education]
### Hi everyone,

I sent my request to the student package but it was rejected because "Unfortunately, we weren't able to approve your educational discount request for francymak." 

Now I am trying to submit a new one but the "Continue" button is grayed out.

![CleanShot 2023-12-18 at 09 19 59@2x](https://github.com/community/community/assets/75610581/272eda25-9a8e-4067-8e70-9c92f543904e)

Other than that, I sent my student card and they didn't accept it, now I don't understand what to send to get it verified, also because the email has the domain verified (as you can see from the screenshot).

**Comment by FrancyMak** [comment]
![CleanShot 2023-12-18 at 09 26 14@2x](https://github.com/community/community/assets/75610581/329a52ec-0647-4bda-bcf8-cd2467678156)

**Comment by manik-007** [comment]
Hello @FrancyMak 

The reason for the continue button grayed out is that you didn't fill the "How do you plan to use GitHub" section which is compulsory.

I hope this helps and if have any further queries then feel free to ask. I'll be there to answer :)
